Makefile

```diff
@@ -7,6 +7,16 @@ RDMD_FLAGS=--force --build-only --compiler=$(D_COMPILER) $(D_FLAGS)
 
 PLATFORM := $(shell uname -s)
 
+LDMD=ldmd2
+
+# force this to run on every make. make eats newlines.
+LDC_VERSION_STRING := $(shell $(LDMD) --version | head -1)
```

Comment:
`ldmd2 --version` calls `ldc2 -version` under the hood, and the latter works even for older versions of LDC.